waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -24,6 +24,9 @@ type PostgresDriver* = ref object of ArchiveDriver
   writeConnPool: PgAsyncPool
   readConnPool: PgAsyncPool
 
+method getDbType*(s: PostgresDriver): string =
+    return "postgres"
```

Comment:
Would it be an overkill using an `enum` for this?

Reply:
based on the use-case yes, since we are not doing a case type conditional logic.